tests/integration.sh

```
@@ -128,7 +140,8 @@ echo "--------------------------------------------------"
 if [[ $DRONE_BRANCH == "master" || $DRONE_BRANCH == *"refs/tags"* || $DRONE_BRANCH == "release-"* ]]; then
     if [[ $DRONE_BUILD_EVENT == "push" ]]; then
         package_offline_installer 
-        upload_latest_build=true     
+        upload_latest_build=true
+        publishImage     
```

please move it to line #203, in case fail early and